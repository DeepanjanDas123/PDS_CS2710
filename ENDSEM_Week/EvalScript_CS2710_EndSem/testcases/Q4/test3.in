GenerateLengthLimitedHuffman
qgbinekjq
3
Quit
