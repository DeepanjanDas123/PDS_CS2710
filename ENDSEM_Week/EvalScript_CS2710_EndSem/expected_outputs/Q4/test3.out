b 000
e 001
g 010
i 011
j 100
k 101
n 110
q 111
cost: 27
